packages/plugin-ext/src/main/browser/documents-main.ts

```diff
@@ -169,7 +169,7 @@ export class DocumentsMainImpl implements DocumentsMain, Disposable {
     async $tryCreateDocument(options?: { language?: string; content?: string; }): Promise<UriComponents> {
         const language = options && options.language;
         const content = options && options.content;
-        const resource = createUntitledResource(content, language);
+        const resource = await createUntitledResource(content, language);
```

Comment:
It could be replaced by `UntitledResourceResolver.create(content, language)` instead of calling global function?

Reply:
@ShimonBenYair will change to call UntitledResourceResolver via rpc proxy

Reply:
Don't we have `Container` in the constructor from which we can get a resolver with `container.get(UntitledResourceResolver)`? Not sure what you mean by balling via rpc proxy.